assets/stylesheets/index.scss

```
@@ -6,16 +6,19 @@
 // Base
 @import 'base/button';
 @import 'base/form';
-@import 'base/nav';
 @import 'base/table';
 @import 'base/typography';
 
 // Components
 @import 'components/alert';
+@import 'components/search-result';
+@import 'components/search-result-list';
 
 // Layouts
 @import 'layouts/application';
 @import 'layouts/authentication';
 
 // Screens
-@import 'screens/keyword.scss';
+@import 'screens/keyword_result';
```

Use hyphens for file names:

```suggestion
@import 'screens/keyword-result';
```
Same issue for `'screens/not_found';` -> `'screens/not-found';

---

Good catch! Fixed in 5e3a7d09fca8378612cee9a70a962b78c6f12e93